libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -42,7 +42,7 @@ public class IpvSessionItem implements PersistenceItem {
     private long ttl;
     private String emailAddress;
     private ReverificationStatus reverificationStatus;
-    private List<String> stateStack = new ArrayList<>();
+    @Builder.Default private List<String> stateStack = new ArrayList<>();
```

Comment:
This is fixing a compilation warning - lombok didn't like the mix of having a builder and having the field already initialised.

Reply:
(Fine - although we basically never actually use the builder)

Reply:
I think I introduced in pretty recently which is why there probably isn't much adoption of it yet - it was useful for tests